src/main/java/SYSC4806/Model/Course.java

```diff
@@ -0,0 +1,74 @@
+package SYSC4806.Model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.util.List;
+
+/**
+ * Model class used to map between learning outcomes, programs and courses.
```

Comment:
Although this comment describes the relationships between models, it could be a bit more specific. E.g. "Course is a model class that represents a course a student can take. A course belongs to many programs and a student should acquire certain learning outcomes by taking this course". --> that can be shortened or said in a more technical matter, but that's just an example of what I mean. 

*ditto for the rest of the model description comments*